### PR TITLE
first

### PR DESCRIPTION
**Pull request by MistahSanta**
small fix

### CODE REVIEW

File: pages/api/graphql/index.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ import AdditionalProfileResolver from 'lib/graphql/resolvers/AdditionalProfileRe
 
 
 if (process.env.NODE_ENV !== "development") {
-  applyResolversEnhanceMap(resolversEnhanceMap);
+applyResolversEnhanceMap(resolversEnhanceMap);
```
Comment (DedsecKnight) [inline pages/api/graphql/index.ts]: Can we undo this?

File: components/vanity/VanityPopUp.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import React, { Fragment } from 'react';
+import { VanityLink } from '@generated/type-graphql';
+import { Dialog, Transition } from "@headlessui/react";
+import { motion } from 'framer-motion';
+import styled from 'styled-components';
+import CloseIcon from 'icons/CloseIcon';
+
+
+
+type vanityPopUpProps = {
```
Comment (DedsecKnight) [inline components/vanity/VanityPopUp.tsx]: Rewrite this to `VanityPopUpProps` instead of `vanityPopUpProps`

**Review by DedsecKnight — CHANGES_REQUESTED** [review]
just a few nit, but everything else looks good to me.

File: components/vanity/VanityForm.tsx [hunk 3]
```diff
@@ -27,26 +30,48 @@ export default function VanityForm() {
...
+
   return (
-    <div className="w-full grid place-items-center px-3">
+    <div className="w-full grid place-items-center px-3 relative">
+
       <div className="flex flex-col p-10 place-items-center">
         <div className="text-3xl font-semibold text-gray-100">create a vanity link</div>
       </div>
-      <div className="flex flex-col md:flex-row-reverse w-full md:w-[50%]">
+      <div className="flex flex-col md:flex-row-reverse w-full md:w-[50%]">\
```
Comment (DedsecKnight) [inline components/vanity/VanityForm.tsx]: Is the `\` intentional?

Comment (MistahSanta) [reply components/vanity/VanityForm.tsx]: oh good catch!

**Review by DedsecKnight — APPROVED** [review]
:+1:

**Pull request merged**